proto/replicate_mode.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+package replicate_mode;
+
+// The replicate status sync from PD to TiKV.
+message ReplicateStatus {
+    enum Mode {
+        MAJORITY = 0;  // the standard mode. Replicate logs to majority peer.
+        MS_AUTOSYNC = 1; // master-slave mode. Replicate logs among 2 DCs.
```

Comment:
How about "PS_AUTO"? "PS" stands for primary and secondary. Or "Active-Active".

Reply:
seems PS is harder for reader to guess its meaning. I'm thinking maybe just AUTOSYNC